app/src/main/res/layout/activity_clip_image.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Same as in the other PR, can we please have a short description of what the expected output is? Also, please auto-format the file so that the attributes are ordered. Thank you!